python/nvfuser_direct/__init__.py

```diff
@@ -379,3 +379,66 @@ def from_pytorch(self, tensor, static_sizes=False):
         )
         self.fusion.add_input(tv)
         return tv
+
+    def validate(
+        self,
+        inputs: list[torch.Tensor],
+        reference_outputs: list[torch.Tensor] = None,
```

Comment:
nitpick is all I do :wink: :
```suggestion
        reference_outputs: Optional[list[torch.Tensor]] = None,
```